scripts/server

```diff
@@ -12,17 +12,26 @@ Starts servers using docker-compose
 "
 }
 
-
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
     if [ "${1:-}" = "--help" ]
     then
         usage
+    elif [ "$1" = "--no-airflow" ]
```

Comment:
I should have defaulted this, since the script now breaks if you don't have anything in the first parameter slot. Lemme know if you want me to make this tiny change or if you want to.